http_parser.c

```diff
@@ -406,6 +397,118 @@ enum http_host_state
   , s_http_host_port
 };
 
+
+static const char *method_strings[] = {
+    [HTTP_DELETE] = "DELETE",
+    [HTTP_GET] = "GET",
+    [HTTP_HEAD] = "HEAD",
+    [HTTP_POST] = "POST",
+    [HTTP_PUT] = "PUT",
+    [HTTP_CONNECT] = "CONNECT",
+    [HTTP_OPTIONS] = "OPTIONS",
+    [HTTP_TRACE] = "TRACE",
+    [HTTP_COPY] = "COPY",
+    [HTTP_LOCK] = "LOCK",
+    [HTTP_MKCOL] = "MKCOL",
+    [HTTP_MOVE] = "MOVE",
+    [HTTP_PROPFIND] = "PROPFIND",
+    [HTTP_PROPPATCH] = "PROPPATCH",
+    [HTTP_SEARCH] = "SEARCH",
+    [HTTP_UNLOCK] = "UNLOCK",
+    [HTTP_BIND] = "BIND",
+    [HTTP_REBIND] = "REBIND",
+    [HTTP_UNBIND] = "UNBIND",
+    [HTTP_ACL] = "ACL",
+    [HTTP_REPORT] = "REPORT",
+    [HTTP_MKACTIVITY] = "MKACTIVITY",
+    [HTTP_CHECKOUT] = "CHECKOUT",
+    [HTTP_MERGE] = "MERGE",
+    [HTTP_MSEARCH] = "M-SEARCH",
+    [HTTP_NOTIFY] = "NOTIFY",
+    [HTTP_SUBSCRIBE] = "SUBSCRIBE",
+    [HTTP_UNSUBSCRIBE] = "UNSUBSCRIBE",
+    [HTTP_PATCH] = "PATCH",
+    [HTTP_PURGE] = "PURGE",
+    [HTTP_MKCALENDAR] = "MKCALENDAR",
+    [HTTP_LINK] = "LINK",
+    [HTTP_UNLINK] = "UNLINK"
+};
+
+
+static const char *errno_name[] = {
```

Comment:
These symbols are globally available everywhere, where `http_parser.h` is included. I'm not sure it is a good idea to declare it with such unprefixed names.